cli/lib/list.go

```diff
@@ -94,13 +93,17 @@ func runListCommand(cmd *cobra.Command, args []string) {
 		libs = []*rpc.InstalledLibrary{}
 	}
 
-	feedback.PrintResult(installedResult{libs})
+	feedback.PrintResult(installedResult{
+		onlyUpdates:   updatable,
+		installedLibs: libs,
+	})
```

Comment:
It is because previously `updatable` was a global variable in the package, now it's local and the method `installedResult.String()` cannot access it anymore, it must be passed through the struct.

This is also a better implementation because the `installedResult.String()` method should return the string representation of the underlying `installedResult` independently from the value of an external global variable.